Dockerfile

```diff
@@ -6,12 +6,12 @@ WORKDIR /code
 RUN apk add --no-cache gcc musl-dev libffi-dev openssl-dev rust cargo
 
 RUN pip --no-input --no-cache-dir install --upgrade pip wheel
-RUN pip --no-input --no-cache-dir install 'poetry==1.1.6'
+RUN pip --no-input --no-cache-dir install 'poetry==1.2.0b2'
```

Comment:
I think it might be better to not pin a specific version of poetry but instead use the latest version, like we do with pip/wheel. At least that's how it is on `hathor-core`, mostly because we had issues with things stopping to work for not using the latest poetry, at least that happens more frequently than it breaking for using the latest version (which I believe has never happened).

Reply:
I will try `'poetry>=1.2.0b2'`, like you suggested on Slack